### ISSUE TITLE
[Improvement] /chc purge {multiple players}

### ISSUE THREAD

**Issue by LogGits** [kangarko/ChatControl#900]
So I noticed when trolls come on, they come in groups. This requires staffer to utilise `/chc purge {player}` for each player. This means players will notice multiple chat flashes. Is it possible to turn `/chc purge {player}` into `/chc purge {player...}` whereby you can specify many players e.g:
```
/chc purge Larry Garry Sarah Susan
```
In doing so, this would yield one flash update to chat and be much easier for staffers to use (if it uses tabcomplete for all args). This benefits alot of servers because it makes the /chc purge command easier and faster to use.

**What would happen if we won't implement this feature? How's your server being affected?**
Will have to use existing method of having multiple flashing chats and cause alot of confusion.

**Comment by ElBananaa** [comment]
As a server owner, I think it's a great idea, but it also brings concerns regarding performances. There are already known issues with the performances of the plugin. While Kangarko tries to improve the plugin on each update, I feel like this would be making a step forward when it comes to features, but 2 steps backward when it comes to performances, because it would mean the plugin would have to potentially remove a lot of messages at the same time, which seems to be extremely heavy.

Anyways, I'm certainly not the best person to talk about this, so let's wait for Kangarko :)

**Comment by LogGits** [comment]
Can likely be done async and would be more performant then using the command multiple times simply because it would be the same amount of calcs but using the command multiple times will cause multiple chat refreshes which would cause flickering chat and send alot of unnecessary packets to the player. Using one argument in this suggestion would yield the same amout of computations as the current system. E.g. for loop multiple times and one comparison is the same as one for loop and multiple comparisons.

 Tldr: will be more performant then existing usage in all cases except for purging 1 player and in that case the computation would be the same as the existing.

**Comment by kangarko** [comment]
I made this async, so adding support for multiple players pose no performance penalty on the main thread, added! :)
